rust/build.gradle.kts

```diff
@@ -5,7 +5,7 @@ plugins {
 rust {
     command = "cargo"
 
-    environment = mapOf(Pair("RUSTUP_TOOLCHAIN", "nightly"))
+    environment = mapOf(Pair("RUSTUP_TOOLCHAIN", "stable"))
```

Comment:
This could be [`mapOf("RUSTUP_TOOLCHAIN" to "stable")`](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin/to.html) I think

Reply:
good catch